SimpleFTP/SimpleFTP/FTPClient.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace SimpleFTP;
+
+using System.Net.Sockets;
+
+/// <summary>
+/// Client class.
+/// </summary>
+/// <param name="serverAddress">Host name.</param>
+/// <param name="serverPort">Port.</param>
+public class FTPClient(string serverAddress, int serverPort)
+{
+    private readonly string host = serverAddress;
+    private readonly int port = serverPort;
+
+    /// <summary>
+    /// Listing files in a directory on the server.
+    /// </summary>
+    /// <param name="directoryPath">Path.</param>
+    /// <returns>List of contents if a directory.</returns>
+    public async Task<(string path, bool isDirectory)[]> List(string directoryPath)
```

Comment:
```suggestion
    public async Task<(string Path, bool IsDirectory)[]> List(string directoryPath)
```
стайлгайд требует писать имена переменных в кортежах так